### PR TITLE
[fix bug 1363543] /new variations to support ad campaigns - private, general, competitor

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Three new variations for ad campaigns, based on previous templates.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1363543

## Testing
- Copy pasta errors are likely the biggest error prone thing here

https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/?xv=moreprotection
https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/?xv=workingout
https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/?xv=youdoyou

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

File: media/css/firefox/new/fx-lifestyle/browse-up-to-you.less [hunk 1]
```diff
@@ -5,6 +5,7 @@
 @import '../../../sandstone/lib.less';
 
 #intro {
+    background-color: #868b8f;
```
Comment (alexgibson) [inline media/css/firefox/new/fx-lifestyle/browse-up-to-you.less]: Added some background colors that match the pallet of each image. I noticed while working out in the garden that the pages are unreadable until the images load, which will likely hurt slower connections :/

File: media/css/firefox/new/fx-lifestyle/conformity-not-default.less [hunk 2]
```diff
@@ -14,6 +15,10 @@
         p {
             .font-size(22px);
         }
+
+        .copy {
+            width: 500px;
```
Comment (alexgibson) [inline media/css/firefox/new/fx-lifestyle/conformity-not-default.less]: Had to adjust the width here to accommodate slightly more copy, given that the images are all fixed height :/

Comment (alexgibson) [inline media/css/firefox/new/fx-lifestyle/conformity-not-default.less]: We could totally change these to use `background-size: cover` which would be more flexible, but given that these are in english only I decided against it for now.

File: media/css/firefox/new/fx-lifestyle/browse-up-to-you.less (hunk 1 above)
Comment (jpetto) [inline media/css/firefox/new/fx-lifestyle/browse-up-to-you.less]: Nice addition.

🏋️ 🏡

**Pull request merged**